pkg/operator/controllers/guardrails/policies/gkconstraints/aro-clusterversion-upgradeconfig-deny.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: ARODenyClusterVersionUpgradeConfig
+metadata:
+  name: aro-deny-clusterversion-upgradeconfig-modification
```

Comment:
we want to have the same .metadata.name as the filename of the constraint to keep feature flag name and constraint name consistent, so can you please rename it to aro-clusterversion-upgradeconfig-deny